src/coreclr/src/vm/method.cpp

```diff
@@ -1329,11 +1329,11 @@ ReturnKind MethodDesc::ParseReturnKindFromSig(INDEBUG(bool supportStringConstruc
 
 ReturnKind MethodDesc::GetReturnKind(INDEBUG(bool supportStringConstructors))
 {
-#ifdef HOST_64BIT
+#ifdef TARGET_64BIT
```

Comment:
This ifdef can be just delete (keep the comment though).

Reply:
Fixed. Thank you.